crates/viewer/re_space_view_spatial/Cargo.toml

```diff
@@ -47,6 +47,9 @@ bytemuck.workspace = true
 egui = { workspace = true, features = ["serde"] }
 glam.workspace = true
 hexasphere.workspace = true
+image = { workspace = true, features = [
+  "png",
+] } # For loading embedded images to textures.
```

Comment:
What "embedded images"? Do you mean `Blob`s? If so, why  only `png` and not also `jpeg`?

EDIT: oh I see, it is for the video icon. Maybe "For loading images/icons for the GUI"?